recipes/braggedgemodeling/meta.yaml

```diff
@@ -49,3 +48,7 @@ about:
 
 extra:
   recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+    - LisaSimpson
+    - LandoCalrissian
```

Comment:
These must be populated with real GitHub handles of people who will maintain the recipe.

Reply:
already fixed. thanks